### PR TITLE
[TACHYON-1379] Add docs for all commands in bin/tachyon

### PR DESCRIPTION
**Pull request by RongGu**
I added the full list of the commands in bin/tachyon with detailed explanations of their meanings in  the Developer-Tips.md. It looks as below.
developer-tips

### CODE REVIEW

File: docs/Developer-Tips.md [hunk 1]
```diff
@@ -47,3 +47,112 @@ Then to regenerate the Java code, run
...
+
+Most commands in `bin/tachyon` are for developers. The following table explains the meaning and the
+syntax of each command.
+
+<table class="table table-striped">
+<tr><th>Command</th><th>Args</th><th>Meaning</th></tr>
+<tr>
+  <td>format</td>
+  <td>[-s]</td>
+  <td>Format Tachyon Master and all Workers. The option [-s] indicates that only format when underfs
```
Comment (aaudiber) [inline docs/Developer-Tips.md]: NIT: The option [-s] indicates that the command should only format when the underfs is local and doesn't already exist.

File: docs/Developer-Tips.md [hunk 2]
```diff
@@ -47,3 +47,112 @@ Then to regenerate the Java code, run
...
+  <td>tfs</td>
+  <td>[tfs-commands]</td>
+  <td>Interact with Tachyon in command line style for basic file system operations.
+  See <a href="Command-Line-Interface.html">Command Line</a> for more information.</td>
+</tr>
+<tr>
+  <td>loadufs</td>
+  <td>&lt;TachyonPath&gt; &lt;UfsPath&gt; [ExcludePathPrefixes]</td>
+  <td>Loads files under `UfsPath` to the given `TachyonPath`. `ExcludePathPrefixes` can be a set of
+  prefixes which are separated by ';'. The paths with the prefix in `ExcludePathPrefixes` will not
```
Comment (aaudiber) [inline docs/Developer-Tips.md]: the prefix -> a prefix

File: docs/Developer-Tips.md [hunk 3]
```diff
@@ -47,3 +47,112 @@ Then to regenerate the Java code, run
...
+  <td>None</td>
+  <td>Print Tachyon version.</td>
+</tr>
+<tr>
+  <td>validateConf</td>
+  <td>None</td>
+  <td>Validate Tachyon conf.</td>
+</tr>
+</table>
+
```
Comment (aaudiber) [inline docs/Developer-Tips.md]: NIT: extra newline

File: docs/Developer-Tips.md [hunk 4]
```diff
@@ -47,3 +47,112 @@ Then to regenerate the Java code, run
...
+
+In addition, these commands have different prerequisites. The prerequisite for `format`,
+`formatWorker`, `journalCrashTest`, `readJournal`, `version` and `validateConf` commands is that you
+have already built Tachyon (see [Build Tachyon Master Branch](Building-Tachyon-Master-Branch.html)
+about how to building Tachyon manually). Further, the prerequisite for `tfs`, `loadufs`, `runTest`
```
Comment (aaudiber) [inline docs/Developer-Tips.md]: how to building -> how to build
prerequisite for -> prerequisite for the

File: docs/Developer-Tips.md [hunk 5]
```diff
@@ -47,3 +47,112 @@ Then to regenerate the Java code, run
...
+</tr>
+<tr>
+  <td>formatWorker</td>
+  <td>None</td>
+  <td>Format Tachyon Worker storage on this local node.</td>
+</tr>
+<tr>
+  <td>bootstrap-conf</td>
+  <td>&lt;TACHYON_MASTER_HOSTNAME&gt;</td>
+  <td>Generate the bootstrap config file `tachyon-env.sh` with the specified
```
Comment (aaudiber) [inline docs/Developer-Tips.md]: Based on the image you showed, it looks like the backticks aren't monospacing `tachyon-env`. Is another approach needed when inside a `<td>` element?

**Comment by aaudiber** [comment]
Thanks for this great improvement to developer tooling documentation! I've left some comments.

**Comment by RongGu** [comment]
Thanks for the comments.
Now I use '\<code\>...\</code\>' instead of the backticks in the `<td>` element. And it looks like below:
code-style

File: docs/Developer-Tips.md [hunk 6]
```diff
@@ -47,3 +47,111 @@ Then to regenerate the Java code, run
...
 $ bin/tachyon protoGen
 ```
+
+### Full list of the commands in bin/tachyon
+
+Most commands in `bin/tachyon` are for developers. The following table explains the meaning and the
+syntax of each command.
+
+<table class="table table-striped">
+<tr><th>Command</th><th>Args</th><th>Meaning</th></tr>
```
Comment (yupeng9) [inline docs/Developer-Tips.md]: Maybe `Description` is better than `Meaning`?

**Comment by yupeng9** [comment]
LGTM. Thanks for the doc improvement!

**Comment by apc999** [comment]
+1 LGTM. Thanks for improving the doc!

**Comment by calvinjia** [comment]
LGTM

**Pull request merged**